modules/tensor_mechanics/src/materials/Compute2DFiniteStrain.C

```diff
@@ -37,9 +37,9 @@ Compute2DFiniteStrain::computeProperties()
     RankTwoTensor A((*_grad_disp[0])[_qp], (*_grad_disp[1])[_qp], (*_grad_disp[2])[_qp]); //Deformation gradient
     RankTwoTensor Fbar((*_grad_disp_old[0])[_qp], (*_grad_disp_old[1])[_qp], (*_grad_disp_old[2])[_qp]); //Old Deformation gradient
 
-    // Compute the deformation gradient (2,*) value for plane strain, generalized plane strain, or axisymmetric problems
-    A(2,2) = computeDeformGradZZ();
-    Fbar(2,2) = computeDeformGradZZold();
+    // Compute the displacement gradient (2,2) value for plane strain, generalized plane strain, or axisymmetric problems
```

Comment:
We should discuss the name of this function.  It seems to me like the old name was fine, but you must have had a reason for changing it, so I'd like to hear your thoughts.